resources/lib/ui/utils.py

```diff
@@ -13,6 +14,9 @@ def fetch_sources(sources, dialog, raise_exceptions=False):
     fetched_sources = []
```

Comment:
Corrected how the loop was broken. Passing the variables all the way from default.py is looking like a massive task.